zorg/buildbot/builders/annotated/fuchsia-linux.py

```diff
@@ -41,13 +42,9 @@ def main(argv):
     buildbot_revision = os.environ.get('BUILDBOT_REVISION', 'origin/main')
 
     cwd = os.getcwd()
-    build_dir = os.path.join(cwd, '..', 'llvm-build')
+    build_dir = tempfile.mkdtemp('', 'llvm-build-', cwd)
     source_dir = os.path.join(cwd, '..', 'llvm-project')
 
-    if buildbot_clobber:
-        with step('clean', halt_on_fail=True):
-            util.clean_dir(build_dir)
```

Comment:
I think it might be still worth supporting this, but rather than cleaning the `build_dir`, it should empty out `cwd` by calling `util.clean_dir(cwd)`, but this needs to happen before calling `tempfile.mkdtemp`.

Reply:
We should also update the ccache configuration to place the cache in `cwd` but that can be done in a follow up change.